app/code/Magento/Contact/Api/MailInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Contact module base controller
+ */
+namespace Magento\Contact\Api;
+
+interface MailInterface
+{
+    /**
+     * Send email from contact form
+     *
+     * @param string $replyTo Reply-to email address
+     * @param array $variables Email template variables
```

Comment:
Service Contracts does not allow usage of "array" type, it should be more specific, i.e. "String[]"